cr.c

```diff
@@ -77,6 +78,12 @@ int dill_no_blocking(int val) {
     return old;
 }
 
+static void dill_ctx_timer_handler(int sig, siginfo_t *si, void *uc) {
+    struct dill_ctx_cr *ctx = si->si_value.sival_ptr;
+    int or = timer_getoverrun(ctx->timer);
+    if(or) ctx->do_poll = 1;
```

Comment:
I guess 0 is returned only if you call it before the timer fires which sounds like a legitimate use case.

Anyway the worst thing that could happen here is that under 100% CPU load external events would be checked more frequently than once per second.

Anyway, I don't care much. Keep it or leave it as you wish.